ethcore/src/engines/authority_round/mod.rs

```diff
@@ -1298,8 +1306,13 @@ impl Engine<EthereumMachine> for AuthorityRound {
 				//   likely ignore old reports
 				// - This specific check is only relevant if you're importing (since it checks
 				//   against wall clock)
-				if let Ok((_, set_number)) = self.epoch_set(header) {
-					self.validators.report_benign(header.author(), set_number, header.number());
+				if let Ok((set, set_number)) = self.epoch_set(header) {
+					let me = self.address();
+					if set.contains(header.parent_hash(), &me) {
+						self.validators.report_benign(header.author(), set_number, header.number());
+					} else {
+						debug!(target: "engine", "Not reporting benign misbehaviour (cause: InvalidSeal) because we're not part of the validator set, Own address: {}", me);
```

Comment:
I'd make it a `warn` - cause it means the node is configured as validator `(--engine-signer)`, but it's not on the list and that's most likely some misconfiguration.